orderbook/src/account_balances.rs

```diff
@@ -196,7 +196,7 @@ mod tests {
     #[tokio::test]
     #[ignore]
     async fn mainnet_can_transfer() {
-        let http = create_test_transport("http://127.0.0.1:8545");
+        let http = create_test_transport(&std::env::var("NODE_URL").unwrap());
```

Comment:
nit: why not change the `create_test_transport` to just read from `env` directly instead of reading it at every call site?
```
fn create_test_transport() {
  let url = std::env::var("NODE_URL").unwrap()
  // ...
}
```

Reply:
I felt that create_test_transport should just create the transport without deciding on the url. But I don't mind that way either.

Reply:
Another suggestion would be to:
```
fn create_default_test_transport() {
  create_test_transport(std::env::var("NODE_URL").unwrap_or_else("http://127.0.0.1:8545"));
}
```